src/Host/Client/Impl/BrokerServices/RemoteUriWebService.cs

```diff
@@ -22,17 +23,18 @@ public class RemoteUriWebService : IRemoteUriWebService {
 
         private Uri PostUri { get; }
 
-        public async Task GetResponseAsync(HttpListenerContext context, string localBaseUrl, string remoteBaseUrl, CancellationToken ct) {
+        public async Task GetResponseAsync(HttpListenerContext context, string localBaseUrl, string remoteBaseUrl, string webSocketsScheme, CancellationToken ct) {
             string postUri = null;
             if (context.Request.IsWebSocketRequest) {
-                UriBuilder ub = new UriBuilder(PostUri) { Scheme = "ws" };
+                UriBuilder ub = new UriBuilder(PostUri) { Scheme = webSocketsScheme };
```

Comment:
Shouldn't this just map "http" -> "ws" and "https" -> "wss"? Are there any cases where we want anything different?

If this is to handle user input with "http://" in Workspaces window, I think it would be better to validate that there, and auto-correct it (i.e. if they edit the URL and use "http://", replace it with "https://" right there). Seeing "https" would indicate that connection is secured to the user.